EnglishFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,9 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/77063
+thewincentral.com#?#.td-block-title-wrap > h4[class^="block-title"]:contains(Advertisement)
+thewincentral.com#?#.td-block-title-wrap > h4[class^="block-title"]:contains(ADVERTISEMENT)
```

Comment:
Use Regex in `:contains()`.
`thewincentral.com#?#.td-block-title-wrap > h4[class^="block-title"]:contains(/Advertisement|ADVERTISEMENT/)`

Reply:
Ah yess, thanks. Btw, how do you use it?

Reply:
We use it in such cases when a single match is not enough.

Reply:
@wout-codes Why not `##.td-g-rec`?

Reply:
For what?

Reply:
for leftover on the right

Reply:
I don't think it's a good idea (may cause false positives). If it would not cause false positives, Zloyden would recommended it.

Reply:
I see another elements which contain "rec" in their classnames, all of them are ads, so I conclude that "rec" means "reclama" not "rectangle"

Reply:
Maybe you can additionally add it?